pkg/config/generic_config.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2020 Red Hat, Inc.
```

Comment:
I think the correct way to deal with this is to use `2020-2021` now.

Reply:
This should be a global change across the repo in a new PR. I'll do it separately.